packages/client/modules/userDashboard/components/OrgBilling/BillingForm.tsx

```diff
@@ -121,11 +124,12 @@ const BillingForm = (props: Props) => {
         setIsLoading(false)
         return
       }
+      UpgradeToTeamTierMutation(atmosphere, {orgId}, {onError, onCompleted})
```

Comment:
-1 I'm not a big fan of having client logic responsible of tying `createSubscription` and `upgradeToTeamTier` mutation together. Can we call the latter internally based on a stripe event we get? This would be much more reliable, otherwise there could be a mismatch the other way round, we could have customers paying still on the free tier.

Reply:
Me neither, but for 3D Secure cards, the client needs to confirm the payment once the subscription has been confirmed. See https://stripe.com/docs/issuing/3d-secure?locale=en-GB and https://stackoverflow.com/questions/71631786/stripe-confirmcardpayment-frontend-vs-paymentintent-confirm-backend for more info.